lib/OperationHandler.cpp

```diff
@@ -160,22 +210,13 @@ void OperationHandler::OnHostSignalQualityChange(const GUID& interfaceGuid, unsi
     });
 }
 
-std::vector<WifiNetworkInfo> OperationHandler::GetUserBss()
-{
-    if (m_clientCallbacks.ProvideFakeNetworks)
-    {
-        return m_clientCallbacks.ProvideFakeNetworks();
-    }
-    return {};
-}
-
 void OperationHandler::AddInterface(const std::function<std::unique_ptr<IWlanInterface>()>& wlanInterfaceBuilder)
 {
     m_serializedRunner.Run([this, wlanInterfaceBuilder] {
         auto wlanInterface = wlanInterfaceBuilder();
         const auto& newGuid = wlanInterface->GetGuid();
-        auto foundIt = std::find_if(
-            m_wlanInterfaces.begin(), m_wlanInterfaces.end(), [&](const auto& i) { return i->GetGuid() == newGuid; });
+        auto foundIt =
```

Comment:
`foundIt` never seems to be used. Can you use `std::any_of` instead, then? Also, in either case, I think the const iterator bounds can be used for the search, eg. `std::cbegin(m_wlanInterfaces)` and `std::cend(m_wlanInterfaces)`.

Reply:
Done.
Interestingly, since I was trying to find an item and not check a property about it, I didn't even though about `std::any_of`.